elsie/boxmixin.py

```diff
@@ -193,9 +234,30 @@ def draw(ctx):
         return self._create_simple_box_item(draw)
 
     def polygon(
-        self, points, color=None, bg_color=None, stroke_width=1, stroke_dasharray=None
-    ):
-        """ Draw a polygon """
+        self,
+        points,
+        color: str = None,
+        bg_color: str = None,
+        stroke_width=1,
+        stroke_dasharray: str = None,
+    ) -> "BoxMixin":
+        """
+        Draws a polygon.
+
+        Parameters
+        ----------
+        points: List[(float, float) | value.LazyPoint]
```

Comment:
There are also the following options:
* When it is a pair, it could be anything that accepts .x() and .y() methods (e.g. "100%", or LazyValue), so not only floats
* Pairs could be defined using List not only Tuple